packages/plugin-connector/src/data-sources/build-sources.js

```diff
@@ -266,10 +271,12 @@ const flushSource = async (source, query, result, dataPath, metaPath, batchSize,
 	);
 	// Spinner stop?
 	if (!writtenRows) {
-		logOut(chalk.yellow(`Finished. 0 rows, did not create table`));
+		spinner?.warn?.(chalk.yellow(`Finished. 0 rows, did not create table`)) ??
+			console.warn(chalk.yellow(`Finished. 0 rows, did not create table`));
 		return null;
 	} else {
-		logOut(`Finished. ${writtenRows} rows`);
+		spinner?.succeed(`Finished. ${writtenRows} rows`) ??
```

Comment:
using nullish coalesce like this is weird; assuming `spinner.succeed` returns `undefined`, the console log will execute either way

Reply:
`spinner.succeed` returns an instance of `spinner`